cypress/e2e/checkout/checkout.notes.cy.js

```diff
@@ -27,50 +32,99 @@ describe('Check out - Notes', () => {
+    AgreementsDetails.createNote({ ...note2, checkoutApp: true });
+    // Navigate to "Check out" app
+    cy.visit(TopMenu.checkOutPath);
+  });
+
   after('Deleting created entities', () => {
-    UserEdit.changeServicePointPreferenceViaApi(testData.userId, [servicePoint.id]);
-    ServicePoints.deleteViaApi(servicePoint.id);
+    UserEdit.changeServicePointPreferenceViaApi(testData.userId, [instanceData.servicePoint.id]);
+    ServicePoints.deleteViaApi(instanceData.servicePoint.id);
+    InventoryInstances.deleteInstanceViaApi({
+      instance: instanceData.folioInstances[0],
+      servicePoint: instanceData.servicePoint,
+      shouldCheckIn: true,
+    });
     Users.deleteViaApi(testData.userId);
   });
 
   it(
-    'C356781: Verify that all notes assigned to user pop up when user scan patron card (“Delete” option) (Spitfire)',
-    { tags: [TestTypes.criticalPath, DevTeams.spitfire] },
+    'C356781: Verify that all notes assigned to user pop up when user scan patron card (“Delete” option) (Spitfire) (TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.spitfire, Parallelization.nonParallel] },
     () => {
-      const note1 = { title: 'Note 1', details: 'This is Note 1' };
-      const note2 = { title: 'Note 2', details: 'This is Note 2' };
-
-      UsersSearchPane.searchByUsername(testData.username);
-      UsersSearchPane.waitLoading();
-      usersCard.openNotesSection();
-      AgreementsDetails.createNote({ ...note1, checkoutApp: true });
-      usersCard.openNotesSection();
-      AgreementsDetails.createNote({ ...note2, checkoutApp: true });
-      cy.visit(TopMenu.checkOutPath);
+      // Fill in user barcode number in the input field at "Scan patron card" pane → Click "Enter" button.
       CheckOutActions.checkOutUser(testData.barcode);
+      // Modal window "Note for patron" with the Note 1 is displayed.
       CheckOutActions.checkUserNote(note1);
+      // Click on the "Delete note" button.
       CheckOutActions.deleteNote();
+      // Modal window "Note for patron" with the Note 2 is displayed
       CheckOutActions.checkUserNote(note2);
+      // Click on the "Delete note" button.
       CheckOutActions.deleteNote();
+      // Open user Details
       cy.visit(TopMenu.usersPath);
       UsersSearchPane.waitLoading();
       UsersSearchPane.searchByUsername(testData.username);
       UsersSearchPane.waitLoading();
+      // Scroll down to "Notes" accordion button and click on it.
       usersCard.openNotesSection();
+      // "Notes" accordion button expanded and the message "No notes found" displayed.
       cy.expect(HTML(including('No notes found')).exists());
     },
   );
+
+  it(
+    'C380512: Verify that all notes assigned to user pop up when user scan patron card (“Close” option) (Spitfire) (TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.spitfire, Parallelization.nonParallel] },
+    () => {
+      // Fill in user barcode number in the input field at "Scan patron card" pane → Click "Enter" button.
```

Comment:
Is it necessary to add comment before each action.

Reply:
useful for AI prompts refining